styleguide/_themes/derek/scss/components/mainnav.scss

```diff
@@ -0,0 +1,371 @@
+}
+
+.navbar-footerLink {
+  @extend h6;
+  color: $white;
+}
+
+//make sure menu aligns with right side of buttons in desktop
+.navbar-menu {
+  float: right;
+  width: auto;
+}
+
+.navbar-dropDownMenu {
+  background: $medium-dark-gray;
+  border: 0;
+  border-radius: 0;
+  min-width: 230px;
+  padding: 0;
+}
+
+.navbar-dropDownItem {
+  position: relative;
+
+  &:hover {
+    background-color: $medium-gray;
+  }
+
+}
+
+.navbar-dropDownSub {
+  position: relative;
+}
+
+.navbar-tertiary-dropDownMenu {
+  background-color: $medium-gray;
+  border: 0;
+  box-shadow: none;
+  left: 100%;
+  margin: 0;
+  min-width: auto;
+  padding: 0;
+  position: absolute;
+  top: 0;
+}
+
+.navbar-tertiary-dropDownItem {
+
+  &:hover {
+    background-color: $light-gray;
+  }
+
+}
+​
+.navbar-dropDownSub {
+
+  &:hover > .navbar-dropDownMenu {
+    display: block;
+  }
+
+}
+
+.navbar-dropDownSub > .navbar-dropDown-trigger {
+
+  &::after {
+    content: '\f0da';
+    float: right;
+    font-family: FontAwesome;
+  }
+
+}
+
+.navbar-brand {
+  background-color: $medium-dark-gray;
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/navigation/rackspace-logo-tagline-simplified.svg');
+  background-position: 10px 14px;
+  background-repeat: no-repeat;
+  background-size: 140px 36px;
+  height: 63px;
+  position: relative;
+  width: 160px;
+}
+
+.navbar-search {
+  background-color: $medium-dark-gray;
+  border-top: 1px solid $dark-gray;
+  display: none;
+  overflow: auto;
+  padding: 10px;
+}
+
+.navbar-searchContainer {
+  float: right;
+  width: 300px;
+}
+
+.navbar-phoneicon,
+.navbar-chaticon,
+.navbar-hamburger,
+.navbar-searchicon {
+  text-align: center;
+
+  &::after {
+    color: $white;
+    display: none;
+    font-family: FontAwesome;
+    padding: 23px 0;
+  }
+
+}
+
+.navbar-hamburger {
+  &::after {
+    content: '\f00d';
+    font-size: 5.5vw;
+  }
+}
+
+.navbar-hamburger.collapsed {
+  &::after {
+    content: '\f0c9';
+  }
+}
+
+.navbar-phoneicon {
+  float: right;
+
+  &::after {
+    content: '\f10b';
+    font-size: 7.5vw;
+  }
+
+}
+
+.navbar-chaticon {
+  float: right;
+
+  &::after {
+    content: '\f086';
+    font-size: 6.5vw;
+  }
+
+}
+
+.navbar-searchicon {
+  float: right;
+  width: 50px;
+
+  &::after {
+    content: '\f002';
+    display: inline-block;
+    font-size: 1vw;
+    margin-right: 25px;
+  }
+
+}
+
+.navbar-activeArrow {
+  border-bottom: 10px solid $medium-dark-gray;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  bottom: -1px;
+  display: none;
+  margin: auto;
+  position: absolute;
+  width: 0;
+}
+
+.navbar-searchInput {
+  border: 1px solid $light-gray;
+  outline: none;
+  padding: 3px;
+  width: 80%;
+}
+
+.navbar-searchButton {
+  float: right;
+  margin: 0;
+  opacity: .7;
+  padding: 6.5px;
+  width: 20%;
+}
+
+
+//main nav show dropdowns on hover on desktop
+@media only screen and (min-width: $screen-md) {
```

Comment:
`$screen-md` (and all other size screen vars) are deprecated as of bootstrap 3.0.1. We should start using `$screen-md-min` and `$screen-md-max` instead. 

`max-width:` media queries will always be used with `$screen-*-max`
`min-width:` media queries will always be used with `$screen-*-min`

That makes the code much more readable and will help prevent one-off errors right at breakpoints.